indigo/distribution.yaml

```diff
@@ -11131,6 +11131,23 @@ repositories:
       url: https://github.com/g/roboteq.git
       version: master
     status: maintained
+  roboteq_diff_driver:
+    doc:
+      type: git
+      url: https://github.com/ecostech/roboteq_diff_driver.git
+      version: master
+    release:
+      packages:
+      - roboteq_diff_driver
+      - roboteq_diff_msgs
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/ecostech/roboteq_diff_driver.git
```

Comment:
You should not use the same repository for the release but use a separate release repository (see [Bloom](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease) instructions.

Same below.

Reply:
Sorry, first time adding packages.  I have removed the release parts for now.